Cache/Hashing.lean

```diff
@@ -34,7 +34,7 @@ def HashMemo.filterByFilePaths (hashMemo : HashMemo) (filePaths : List FilePath)
   for filePath in filePaths do
     if hashMemo.hashMap.contains filePath then
       hashMap := insertDeps hashMap filePath hashMemo
-    else throw $ IO.userError s!"No match for {filePath}"
+    else throw $ IO.userError s!"{filePath} is not covered by the olean cache."
```

Comment:
does this even need to be a `throw`? It can just be a `println!` and continue on without it, since it seems it will still usefully return other dependencies